zh/examples.md

```diff
@@ -0,0 +1,49 @@
+# 项目实战
```

Comment:
It is recommended to separate **plugins** into a single category，contains sections like below .
- How to Use Plugin and Introduction to commonly used plugins 
- With significant links to the plugin search page
- How to write a  Plugin

add ** recipe**  or **integrate** (like CRA Vue-cli )  section in example